management-api-agent-common/src/main/java/com/datastax/mgmtapi/NodeOpsProvider.java

```diff
@@ -733,10 +735,11 @@ public void clearSnapshots(
   }
 
   @Rpc(name = "repair")
-  public void repair(
+  public String repair(
       @RpcParam(name = "keyspaceName") String keyspace,
       @RpcParam(name = "tables") List<String> tables,
-      @RpcParam(name = "full") Boolean full)
+      @RpcParam(name = "full") Boolean full,
+      @RpcParam(name = "async") boolean notifications)
```

Comment:
Question:
Why name the parameter "async" but call the variable "notifications"? It would be less confusing if they were named the same or similarly.

Reply:
Changed. Although I'm not sure if the RpcParam's name is used anywhere?